feature/experimental/backup_nhg/otg_tests/backup_nhg_test/backup_nhg_test.go

```diff
@@ -281,19 +307,24 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 // entry.
 func (a *testArgs) configureBackupNextHopGroup(t *testing.T, del bool) {
 	t.Logf("Adding NH %d with atePort2 via gRIBI", nh1ID)
-	a.client.AddNH(t, nh1ID, atePort2.IPv4, deviations.DefaultNetworkInstance(a.dut), fluent.InstalledInFIB)
+	nh1, op1 := gribi.NHEntry(nh1ID, atePort2.IPv4, deviations.DefaultNetworkInstance(a.dut), fluent.InstalledInFIB)
 	t.Logf("Adding NH %d with atePort3 and NHGs %d, %d via gRIBI", nh2ID, nhg1ID, nhg2ID)
-	a.client.AddNH(t, nh2ID, atePort3.IPv4, deviations.DefaultNetworkInstance(a.dut), fluent.InstalledInFIB)
-	a.client.AddNHG(t, nhg1ID, map[uint64]uint64{nh1ID: 100}, deviations.DefaultNetworkInstance(a.dut), fluent.InstalledInFIB)
-	a.client.AddNHG(t, nhg2ID, map[uint64]uint64{nh2ID: 100}, deviations.DefaultNetworkInstance(a.dut), fluent.InstalledInFIB)
+	nh2, op2 := gribi.NHEntry(nh2ID, atePort3.IPv4, deviations.DefaultNetworkInstance(a.dut), fluent.InstalledInFIB)
+	nhg1, op3 := gribi.NHGEntry(nhg1ID, map[uint64]uint64{nh1ID: 100}, deviations.DefaultNetworkInstance(a.dut), fluent.InstalledInFIB)
+	nhg2, op4 := gribi.NHGEntry(nhg2ID, map[uint64]uint64{nh2ID: 100}, deviations.DefaultNetworkInstance(a.dut), fluent.InstalledInFIB)
+	a.client.AddEntries(t, []fluent.GRIBIEntry{nh1, nh2, nhg1, nhg2}, []*client.OpResult{op1, op2, op3, op4})
 	t.Logf("Adding an IPv4Entry for %s via gRIBI", nhip)
 	a.client.AddIPv4(t, nhip+"/"+mask, nhg1ID, deviations.DefaultNetworkInstance(a.dut), deviations.DefaultNetworkInstance(a.dut), fluent.InstalledInFIB)
 	t.Logf("Adding NH %d in VRF-B via gRIBI", nh100ID)
-	a.client.AddNH(t, nh100ID, "VRFOnly", deviations.DefaultNetworkInstance(a.dut), fluent.InstalledInFIB, &gribi.NHOptions{VrfName: vrfB})
+	nh100, op5 := gribi.NHEntry(nh100ID, "VRFOnly", deviations.DefaultNetworkInstance(a.dut), fluent.InstalledInFIB, &gribi.NHOptions{VrfName: vrfB})
+	if deviations.BackupNHGRequiresVrfWithDecap(a.dut) {
```

Comment:
I would think that this is a necessary validation for all vendors. Any reason why went to exercise this usecase only with a deviation ?

Reply:
For other vendors it is still `NH --> VRF`, for Arista we are exercising `NH --> {VRF + Decap}`

Reply:
We don't want to change the test case.